lib/3scale_toolbox/commands/remote_command.rb

```diff
@@ -0,0 +1,61 @@
+require 'cri'
+require '3scale_toolbox/base_command'
+require '3scale_toolbox/commands/remote_command/remote_add'
+require '3scale_toolbox/commands/remote_command/remote_remove'
+require '3scale_toolbox/commands/remote_command/remote_rename'
+
+module ThreeScaleToolbox
+  module Commands
+    module RemoteCommand
+      class RemoteCommand < Cri::CommandRunner
+        include ThreeScaleToolbox::Command
+        def self.command
+          Cri::Command.define do
+            name        'remote'
+            usage       'remote <command> [options]'
+            summary     '3scale CLI remote'
+            description '3scale CLI command to manage your remotes'
+            runner RemoteCommand
+          end
+        end
+
+        def run
+          list_remotes
+        rescue StandardError => e
+          warn e.message
```

Comment:
I think it is nicer to just pass the whole error to the warning, so the error itself can customize how it is going to be serialized: `warn e`.

Reply:
this error handling per command will be deleted